ProcessLib/LIE/SmallDeformation/LocalAssembler/SmallDeformationLocalAssemblerMatrixNearFracture-impl.h

```diff
@@ -208,15 +208,25 @@ assembleWithJacobian(
 
         auto& eps = ip_data._eps;
         auto& sigma = ip_data._sigma;
-        auto& C = ip_data._C;
-        auto& material_state_variables = *ip_data._material_state_variables;
+        auto& state = ip_data._material_state_variables;
 
         eps.noalias() = B * nodal_total_u;
 
-        if (!ip_data._solid_material.computeConstitutiveRelation(
+        std::unique_ptr<KelvinMatrixType<DisplacementDim>> C;
```

Comment:
Actually this whole business does not look any better than just passing the out-parameters by non-const reference.